src/bin/polykey-agent.ts

```diff
@@ -114,8 +114,8 @@ async function main(_argv = process.argv): Promise<number> {
     recoveryCode: pkAgent.keyRing.recoveryCode,
     pid: process.pid,
     nodeId: nodesUtils.encodeNodeId(pkAgent.keyRing.getNodeId()),
-    clientHost: pkAgent.grpcServerClient.getHost(),
-    clientPort: pkAgent.grpcServerClient.getPort(),
+    clientHost: pkAgent.webSocketServer.host,
+    clientPort: pkAgent.webSocketServer.port,
```

Comment:
They weren't getters before, just normal methods. I made them getters since the host and port is updated every time you start. A `readonly` can only be set during construction so I couldn't update it every time we started. 

So we have a getter here. The only alternative is to change it to match the`getHost()` style.

Reply:
I think of `getHost` as getters. The `get host` is not a getter. It's a magic method overloading the `.host`.

Reply:
Sure, but the technical term for the magic methods overriding `.host` are getters and setters aren't they?

Is there a change to be made here?

Reply:
The reason to use `.getHost()` like I said is for the ability to use `_host` and make that 

```
class X {
  // this means x is readable and immutable
  readonly x;
  // this means y is readable and mutable
  y;
  _z
  // this means z is readable, but internally mutable, but externally immutable
  get z () {
    return this._z;
  }
  // why use getA over the above?
  // supposed to mean that the A property is initialised after construction, or has specialised construction
  getA() {

  }
}
```

The `host` and `port` are not directly set before construction, they occur after an asynchronous start. That's why I always make them `getHost` and `getPort` to distinguish them from static properties. We've also talked about this distinction before.

Reply:
Ok, This has been fixed.